src/DataViews/include/DataViews/AppInterface.h

```diff
@@ -106,8 +106,12 @@ class AppInterface {
 
   virtual void EnableFrameTrack(const orbit_client_data::FunctionInfo& function) = 0;
   virtual void DisableFrameTrack(const orbit_client_data::FunctionInfo& function) = 0;
-  virtual void AddFrameTrack(const orbit_client_data::FunctionInfo& function) = 0;
-  virtual void RemoveFrameTrack(const orbit_client_data::FunctionInfo& function) = 0;
+
+  enum class AddFrameTrackResult { kAborted, kVisible, kInvisible };
+  enum class RemoveFrameTrackResult { kAborted, kSuccess };
```

Comment:
Pers pref: I don't think `kAborted` is perfect here. It sounds to me like a user action happened (but I also might be wrong). I would prefer something like `kError`, `kInternalError`, `kNotPossible` 

How I think about this: I think about the comment I would write down describing what that does, something like
```
// This function returns ??? when it was not possible to add a frametrack. 
AddFrameTrackResult AddFrametrack(...);
```

Reply:
Thanks for renaming suggestion!

I'm not sure about `kError` and `kInternalError` as it sounds something wrong happens here. But in our UI, it is normal that a user hook and unhook a function from the functions view before taking a capture.

I think `kNotPossible` sounds good! I will take it! Thanks!